core/db_models/EEM_Form_Element.model.php

```diff
@@ -148,16 +149,23 @@ protected function __construct(FormStatus $form_status, InputTypes $input_types,
                         sprintf(
                             /* translators: 1 class name */
                             __(
-                                'Whether form input is active, archived, trashed, or used as a default on new forms. Values correspond to the %s constants.',
+                                'Whether form element is active, archived, trashed, or used as a default on new forms. Values correspond to the %s constants.',
                                 'event_espresso'
                             ),
-                            'EEM_Form_Element::STATUS_*'
+                            'EventEspresso\core\services\form\meta\FormStatus'
                         )
                     ),
                     false,
                     FormStatus::ACTIVE,
                     $form_status->validStatusOptions()
                 ),
+                'FIN_type'    => new EE_Enum_Text_Field(
+                    'FIN_type',
+                    esc_html__('Form element type.', 'event_espresso'),
+                    false,
+                    false,
+                    $element_types->validTypeOptions()
+                ),
```

Comment:
i had removed this field because technically it counts as an "attribute" 
but this is probably a good idea to keep it separate.
will make things easier this way